runtime/doc/filetype.txt

```diff
@@ -544,6 +544,8 @@ CTRL-T                    Jump back to the location that the manpage was
                           opened from.
 q                         :quit if invoked as $MANPAGER, otherwise :close.
 
+To see the table of contents, type <M-]>.
```

Comment:
We should make `META-foo` and `ALT-foo` highlight. In `runtime/syntax/help.vim`:

```vim
syn match helpSpecial>  >       "CTRL-."
syn match helpSpecial>  >       "META-."
syn match helpSpecial>  >       "ALT-."
```